src/components/FilterTags/FilterTags.js

```diff
@@ -87,30 +88,89 @@ const FilterTags = () => {
     };
   }, []);
 
-  const handleFilterClick = () => {
-    console.log("Filter clicked");
+  /**
+  Apply a given filter
+  @param {object} e The event object
+  @param {string} filter The name of the filter
+   @param {string} filterBy The name of the filter group
+*/
+  const addActiveFilter = filter => {
+    setActiveFilters(filters => {
+      const updatedFilters = [...filters];
+      if (!updatedFilters.includes(filter)) {
+        updatedFilters.push(filter);
+      }
+      return updatedFilters;
+    });
+  };
+
+  /**
+  Remove a given filter
+  @param {object} e The event object
+  @param {string} filter The name of the filter
+   @param {string} filterBy The name of the filter group
+*/
+  const removeActiveFilter = filter => {
+    setActiveFilters(filters => {
+      const updatedFilters = [...filters];
+      if (updatedFilters.includes(filter)) {
+        const index = updatedFilters.indexOf(filter);
+        updatedFilters.splice(index, 1);
+      }
+      return updatedFilters;
+    });
   };
 
   return (
     <div className="p-filter-tags" ref={node}>
-      <form>
-        <input
-          type="text"
-          placeholder="Filter models"
-          className="p-filter-tags__input"
-          onFocus={() => setfilterPanelVisibility(true)}
-        />
-      </form>
+      <input
+        type="text"
+        className="p-filter-tags__input"
+        onClick={() => setFilterPanelVisibility(!filterPanelVisibility)}
```

Comment:
With this change when you click the input again once it's opened it's closed...Was that intentional? Or should it always be open when the input is in focus? (I feel like it should be the latter)

Reply:
Makes sense that it no longer toggles now that clicking anywhere else plus using the Escape key will close the panel.

Reply:
So lets say I click the input to open the dropdown, I select a button, then want to edit the input, when I click the input again it closes the dropdown. I know this doesn't work right now, but I'd imagine this would be a valid usecase when it does :)

Reply:
Re-read the above, I agree with you ;)

Reply:
Oh, yes.... :-D how embarrassing